### PR TITLE
eat excessive white space in block template

### PR DESCRIPTION
**Pull request by chappjc**
I was hacking on the template while working on a different task and this got a little out of hand, so I'm making a separate PR.

### CODE REVIEW

File: views/block.tmpl [hunk 1]
```diff
@@ -1,362 +1,354 @@
...
-                              </td>
-                              <td class="mono fs15 text-right">{{.Fee}}</td>
-                              <td class="mono fs15 text-right">{{.FormattedSize}}</td>
-                          </tr>
-                          {{end}}
-                          {{end}}
-                      </tbody>
-                  </table>
-              {{end}}
-        </div>
```
Comment (chappjc) [inline views/block.tmpl]: this was an unmatched div in the rendered html

File: views/block.tmpl [hunk 2]
```diff
@@ -1,362 +1,354 @@
...
+			</div>
+		</div>
+		<div class="col-24 col-xl-12 secondary-card py-3 px-3 px-xl-4">
+			<div class="h6 d-inline-block my-2 pl-3">Block Details</div>
+			<table class="w-100 fs14 mt-2 details">
+			<tbody>
+				<tr>
+					<td class="text-right font-weight-bold text-nowrap pr-2"
+						><span class="d-none d-sm-inline">Ticket Price</span
+						><span class="d-sm-none">Tkt Price</span>: </td>
```
Comment (chappjc) [inline views/block.tmpl]: This </td> was missing

File: views/block.tmpl [hunk 3]
```diff
@@ -1,362 +1,354 @@
...
-                            <td class="mono fs15 text-right">{{.Fee}}</td>
-                            <td class="mono fs15 text-right">{{.FormattedSize}}</td>
-                        </tr>
-                    {{end}}
-                    </tbody>
-                </table>
-            {{end}}
+		<span class="d-inline-block pt-4 pb-1 h4">Tickets</span>
+	  {{- if not .Tickets}}
+		<table class="table">
```
Comment (buck54321) [inline views/block.tmpl]: A lot of space-to-tab conversion happening here. Is that the intent?

Comment (chappjc) [reply views/block.tmpl]: Yeah, but I've not backed that decision up with post-gzip sizes.  Its a lot fewer chars though.
TBH I'll probably abandon this PR, maybe just grabbing the couple of legit fixes for a different PR.

**Comment by chappjc** [comment]
Superseded by https://github.com/decred/dcrdata/pull/1665

**Pull request closed without merging**